### DISCUSSION TITLE
How to integrate react-virtual with react-beautiful-dnd with window scroll?.

### DISCUSSION THREAD

**Discussion by bipinrajbhar** [TanStack/virtual#229] [Show and tell]
Hi @petersendidit , @piecyk , @mogelbrod , @oskarhane , @shastri9999.

https://codesandbox.io/s/react-virtual-react-beautiful-dnd-integration-forked-3r00z?file=/src/App.js

I am not sure my implementation is correct or not.

**Comment by valjusha** [comment]
Sandbox not found =(
